adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -265,21 +275,21 @@ class VariantContextConverter(dict: Option[SequenceDictionary] = None) extends S
     assert(vc.isBiallelic)
     extractGenotypes(vc, variant, annotations,
       (g: htsjdk.variant.variantcontext.Genotype, b: Genotype.Builder) => {
-        if (g.hasPL) b.setGenotypeLikelihoods(g.getPL.toList.map(p => p: java.lang.Integer))
+        if (g.hasPL) b.setGenotypeLikelihoods(g.getPL.toList.map(p => jFloat(PhredUtils.phredToLogProbability(p))))
```

Comment:
@arahuja On https://github.com/bigdatagenomics/bdg-formats/pull/51, we agreed to move to log probabilities. We were moving to log _success_ probabilities, correct? This is written for log success probabilities, not log error probabilities.